Source/KakapoServer.swift

```diff
@@ -91,11 +103,15 @@ public final class KakapoServer: NSURLProtocol {
     
     /// Start loading the matched requested, the route handler will be called and the returned object will be serialized.
     override public func startLoading() {
-        KakapoServer.routers.filter { $0.canInitWithRequest(request) }.first!.startLoading(self)
+        if let routerIndex = KakapoServer.routers.indexOf({ $0.canInitWithRequest(request) }) {
+            KakapoServer.routers[routerIndex].startLoading(self)
+        }
     }
     
-    /// Not implemented yet, does nothing ATM. https://github.com/devlucky/Kakapo/issues/88
+    /// Stops the loading of the matched request.
     override public func stopLoading() {
-        /* TODO: implement stopLoading for delayed requests https://github.com/devlucky/Kakapo/issues/88 */
+        if let routerIndex = KakapoServer.routers.indexOf({ $0.canInitWithRequest(request) }) {
+            KakapoServer.routers[routerIndex].stopLoading(self)
+        }
```

Comment:
So, if our assumption is right that Foundation is creating a new KakapoServer per request since we return `true` for `canInitWithRequest`: could we, as suggested by @MP0w, just use an internal boolean flag here to later not start loading on `startLoading`. 

That sounds like the most reasonable and simple approach for now, though we can discuss if it's the best one. But it will, almost 100% sure, prevent to modify the `Router`class at all right?